engines/bastion_katello/app/assets/javascripts/bastion_katello/subscriptions/subscription-access-mode.directive.js

```diff
@@ -0,0 +1,21 @@
+/**
+ * @ngdoc directive
+ * @name Bastion.subscriptions:subscriptionAccessMode
+ *
+ * @requires contentAccessMode
+ *
+ * @description
+ *   Directive for the subscription access mode
+ */
+angular.module('Bastion.subscriptions').directive('subscriptionAccessMode',
```

Comment:
Also, why the difference in naming from `contentAccessMode` to `subscriptionAccessMode`.  What is the difference?